go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -244,3 +253,72 @@ func runOvnKube(ctx *cli.Context) error {
 
 	return fmt.Errorf("need to run ovnkube in either master and/or node mode")
 }
+
+// watchForChanges closes stopCh if the configuration file changed.
+func watchForChanges(configPath string, stopCh chan struct{}) error {
+	if configPath == "" {
```

Comment:
Not sure why you need `stopCh`? You are not using it at all inside the function and also you are not passing anything to the channel, but are expecting to receive something at L276

Reply:
@pecameron what about `stopCh`? What is the use of it?